src/Tests/Project/ToolRunner.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import {join} from 'path';
+
+import {ToolRunner} from '../../Project/ToolRunner';
+import {obtainWorkspaceRoot} from '../../Utils/Helpers';
+import {Logger} from '../../Utils/Logger';
+import {MockJob} from '../MockJob';
+
+suite('Project', function() {
+  suite('ToolRunner', function() {
+    const logger = new Logger();
+    let job = new MockJob('mockup');
+    suite('#getOneccPath()', function() {
+      test('returns onecc path as string', function() {
+        let toolRunner = new ToolRunner(logger);
+        let actual = toolRunner.getOneccPath();  // string or undefined
```

Comment:
Shouldn't we also need to check something like the following?
```
assert(actual !== undefined);
```

Reply:
AFAIK, `===` checks object instance and its value so that we can skip it.

Reply:
I mean that `actual === '/usr/share/one/bin/onecc'` checks `actual is not undefined` implicitly.

Reply:
Please note me if I'm wrong. (You know, I'm not good at ts/js.)

Reply:
I am also ts newbie. :-)

I checked the following and found that `?` does the magic.

```typescript
let bar = (n: number) => {
    if (n % 2 === 0) return undefined; 
    else return 'bar';
} 

let foo = bar(2);
console.log((foo === 'bar') || foo?.endsWith('bar'));   // "undefined" is printed
console.log(foo?.endsWith('bar') || (foo === 'bar'));   // "false" is printed

foo = bar(1);
console.log((foo === 'bar') || foo?.endsWith('bar'));   // 'true' is printed
```